fixmydjango/fixmydjango
# Refs #48 - Show "login to comment" message to non logged users

Link to task: https://github.com/fixmydjango/fixmydjango/issues/48

How to test: 
- Enter in any error page not logged and see if the message appears properly
- Click in the link, you should be redirected to the login page 

Link bellow:
![login_to_comment](https://cloud.githubusercontent.com/assets/8882260/23260878/ae34a236-f9b2-11e6-9f21-700db757616f.png)
